consensus/propagation/reactor.go

```diff
@@ -61,11 +76,19 @@ func GetChannels() []*p2p.ChannelDescriptor {
 	return []*p2p.ChannelDescriptor{
 		{
 			// TODO: set better values
-			ID:                  consensus.PropagationChannel,
+			ID:                  WantChannel,
 			Priority:            6,
 			SendQueueCapacity:   100,
 			RecvMessageCapacity: maxMsgSize,
-			MessageType:         &propagation.Message{},
+			MessageType:         &propproto.Message{},
+		},
+		{
+			// TODO: set better values
+			ID:                  DataChannel,
+			Priority:            10,
+			SendQueueCapacity:   100,
```

Comment:
this value needs to be > the worst case scenario, where the proposer has a single peer, an uploads the entire set of haves and block to them. so 500 parts for a 32MB block means ~1000 should be good

Reply:
as we increase block size, we will also need to increase this number, so perhaps the best is to calculate it off of max block size

Reply:
the max block size is defined in app. should we just redefine it here?

Reply:
17a6d05e43e7379c1c043d89bd395641a43e527f